### DISCUSSION TITLE
Feedback requests 2024

### DISCUSSION THREAD

**Discussion by petrvecera** [vsb-vaj/.github#4] [Show and tell]
Because it's really a lot of you, we can't give each one of you personalized feedback on your code in the labs. 

If you would like to receive feedback on your code, please reply in a thread with the link to your repository and file you would like to receive feedback. Don't worry, all labs should be PRIVATE, we have access to them but your classmates can't steal your code. 

Also don't worry about the comments / in most cases you will have correct solution. However we will try to give you as much feedback as possible, for you to learn. A lot of our comments will be just "nitpicking".
>Nitpicking in pull request reviews means offering insight that looks excessive, pedantic, and unimportant. It's a pejorative term. I don't question there are cases where the term applies, but many folks assimilate it to paying attention to the small stuff.

**Comment by Bodkens** [comment]
feedback request https://github.com/vsb-vaj/2024s-lab-01-Bodkens

**Reply by petrvecera** [reply]
You can find feedback here https://github.com/vsb-vaj/2024s-lab-01-Bodkens/pull/1

**Comment by Ajraa** [comment]
Mohl bych prosím dostat feedback k bonusovému tasku v https://github.com/vsb-vaj/2024s-lab-02-Ajraa

**Reply by petrvecera** [reply]
You can find a feedback here https://github.com/vsb-vaj/2024s-lab-02-Ajraa/pull/1
